instant-xml/tests/forward-enum.rs

```diff
@@ -4,19 +4,19 @@ use similar_asserts::assert_eq;
 
 use instant_xml::{from_str, to_string, FromXml, ToXml};
 
-#[derive(Debug, Eq, FromXml, PartialEq, ToXml)]
+#[derive(Debug, PartialEq, FromXml, ToXml)]
 #[xml(forward)]
 enum Foo {
     Bar(Bar),
     Baz(Baz),
 }
 
-#[derive(Clone, Debug, Eq, FromXml, PartialEq, ToXml)]
```

Comment:
Prefer not to add stuff in one commit only to remove it again in a latter one.

(Though thanks for cleaning up the derives, that's a nice touch!)

Reply:
No worries, fixed.